FSI/rigid_flap/OpenFOAM-RigidBody/Solid/rigid_body_solver.cpp

```diff
@@ -0,0 +1,246 @@
+#include <iostream>
```

Comment:
We should also explicitly include `string` here, even if `iostream` includes some parts:

```suggestion
#include <string>
```

Reply:
`iostream` is still required.